### ISSUE TITLE
Support Bootstrap's Toast component

### ISSUE THREAD

**Issue by luisfpg** [valor-software/ng2-bootstrap#5130]
### Feature request:
Add support for Bootstrap 4's toasts: https://getbootstrap.com/docs/4.3/components/toasts/

**Comment by olierxleben** [comment]
I find this feature very interesting. I would like to ask what does the ngx-bs team expect to be contributed? Aswell doc pages and/or cypress tests?

**Comment by dpdhruv** [comment]
This issue sounds like fun , can i please know what exactly needs to be done here?
@ludmilanesvitiy @daniloff200

**Comment by daniloff200** [comment]
@olierxleben  @dpdhruv 

I think, we need some basic toaster functionality here, as described here - https://getbootstrap.com/docs/4.3/components/toasts/

There's pretty big amount of work here, so, I think, for those, who create a PR with this functionality, we can send a full kit of swag, as we have for 5 PRs

**Comment by olierxleben** [comment]
Thx, 

@dpdhruv should we consider working together on it?

**Comment by dpdhruv** [comment]
@olierxleben Yes for sure why not. You can take the lead too because it's my first time diving into open source. There might be time zone issues but we can work on it.

**Comment by dpdhruv** [comment]
Also i read your guidelines for contributing @daniloff200 , should i directly create a new branch from development with appropriate naming convention or should i fork and follow normal procedure?

**Comment by olierxleben** [comment]
I had forked and branched in my fork so far.

**Comment by dpdhruv** [comment]
you started working on it?

**Comment by dpdhruv** [comment]
@daniloff200 Can i work along with @olierxleben in his fork itself ? Will i still be eligible for the giveaways if i do so? 🥇

**Comment by olierxleben** [comment]
No just forked, I had a look around. I have no commitment for the swag. You can have it bro ;)

**Comment by daniloff200** [comment]
Let me answer these questions :)
First of all, yes, it's better to use forks, yes, if you work on that together, we will count that as a pair contribution, no worries on that, you both can get some sweets :) 

Also, @olierxleben , I saw your question in ngx slack, so, you basically can add a new component manually - you can check, how the components added / imported in the project, and make it the same

**Comment by daniloff200** [comment]
Also, don't forget, that you have to not only create it in the library,but also, upgrade the demo

**Comment by dpdhruv** [comment]
> No just forked, I had a look around. I have no commitment for the swag. You can have it bro ;)

I do because it's my first major contribution if i could successfully do it :)

**Comment by dpdhruv** [comment]
Also , i'll join the slack channel so that we can have a better communication and co-ordination @olierxleben

**Comment by olierxleben** [comment]
Just a little update, not that someone thinks I stopped working on it. I was knocked out by the weather here... and sry for any spam you experience now. :P

@dpdhruv I started working on the issue and setup it all for development, if you still want to join, have a look at this branch: https://github.com/olierxleben/ngx-bootstrap/tree/feat-toast where I setup the actual ngx-lib component aswell as the demo project modules. 
I tried to contact you on Slack... just have a look if you have time :)

Screen from my localhost:

<img width="1274" alt="image" src="https://user-images.githubusercontent.com/1403225/66854659-0cbcd680-ef82-11e9-971c-d2500f3f2a7d.png">

**Comment by dpdhruv** [comment]
@olierxleben Sure will have a look at it , was busy couple of weeks here. Sorry , my slack notifications are off i guess.Will have a look today if possible real quick or by the weekend for sure.

**Comment by malcomm** [comment]
Any updates on this? Was just looking getting Toast to work and notice this. Thanks

**Comment by daniloff200** [comment]
@malcomm looks like, it's not :(
